src/style.css

```diff
@@ -2,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  src: url('./fonts/Avenir-Heavy.ttf');
+  src: url('./fonts/Avenir.ttc');
+  font-style: normal;
+  font-weight: normal;
+  color: rgba(61, 78, 97, 1);
```

Comment:
The `color` property is not applicable within the `@font-face` rule. You should remove it as it doesn't affect the font face definition.